### ISSUE TITLE
Frontend is built twice on startup

### ISSUE THREAD

**Issue by Artur-** [vaadin/flow#10083]
### Description of the bug / feature
Running a new Fusion V19 beta 3 project outputs:
```
...
------------------ Starting Frontend compilation. ------------------
...
----------------- Frontend compiled successfully. -----------------
ℹ2021-02-18 14:03:38.257  INFO 49867 --- [nPool-worker-19] com.vaadin.flow.server.DevModeHandler    : Started webpack-dev-server. Time: 13529ms
39m ｢wdm｣: Compiling...
...
----------------- Frontend compiled successfully. -----------------
```
### Expected behavior

Frontend is compiled only once

### Versions:
    - Vaadin / Flow version: 19 beta 3

**Comment by denis-anisimov** [comment]
Duplicate of https://github.com/vaadin/flow/issues/11214
